src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/PInvokeMarshal.Unix.cs

```diff
@@ -60,5 +60,10 @@ public static unsafe int GetCharCount(byte* multiByteStr, int multiByteLen)
             return System.Text.Encoding.UTF8.GetCharCount(multiByteStr, multiByteLen);
         }
         #endregion
+
+        public static unsafe object InitializeCustomMarshaller(RuntimeTypeHandle pParameterType, RuntimeTypeHandle pMarshallerType, string cookie, delegate*<string, object> getInstanceMethod)
```

Comment:
Custom marshallers are not Windows-specific. This should be in platform neutral file.